__tests__/app.test.js

```diff
@@ -208,6 +208,83 @@ describe('GET /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+  test('returns a comment object with required keys and right values', () => {
+    const body = {
+      body: 'test',
+      username: 'butter_bridge',
+    };
+    return request(app)
+      .post('/api/articles/1/comments')
+      .send(body)
+      .expect(200)
+      .then((response) => {
+        const { comment } = response.body;
+        expect(comment).toEqual({
+          comment_id: 19,
+          body: 'test',
+          article_id: 1,
+          author: 'butter_bridge',
+          votes: 0,
+          created_at: new Date(new Date().setMilliseconds(0)).toISOString(),
```

Comment:
If this request took a few seconds to come back then this wouldn't pass, here you could do 
created_at:  expect.any(String) 
and then do some other expect to ensure the string could be a date